pkg/collector/reconcile/route.go

```diff
@@ -1,156 +0,0 @@
-// Copyright The OpenTelemetry Authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package reconcile
-
-import (
-	"context"
-	"fmt"
-
-	"github.com/open-telemetry/opentelemetry-operator/internal/manifests"
-	"github.com/open-telemetry/opentelemetry-operator/internal/manifests/collector"
-
-	routev1 "github.com/openshift/api/route/v1"
-	k8serrors "k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/types"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
-
-	"github.com/open-telemetry/opentelemetry-operator/apis/v1alpha1"
-)
-
-// Routes reconciles the route(s) required for the instance in the current context.
-// TODO: This functionality should be put with the rest of reconciliation logic in the mutate.go
```

Comment:
Why did we keep this? The mutate is already handling routes

Reply:
This was because the old task logic would dynamically add this to its jobs. either way, i think it's probably wholly unused.